docs/book/v1/intro.md

```diff
@@ -14,7 +14,15 @@ return static function (RectorConfig $rectorConfig): void {
 };
 ```
 
-If you want to make renamed class type hint auto import enabled, you may use `SetList::LAMINAS_SERVICEMANGER_40_AUTO_IMPORT` set list, so the `rector.php` config will be as follow:
+Rector works with all class names as fully qualified by default.In the most projects, that's not a desired behavior, because short version with use statement is easier to read.
+
+To import FQCN, configure rector.php with:
+
+```php
+$rectorConfig->importNames();
+```
+
+If you want to make renamed class type hint to use short name with import its fully qualified ot use statement, you may use `SetList::LAMINAS_SERVICEMANGER_40_AUTO_IMPORT` set list that utilize `$rectorConfig->importNames()` on it, so the `rector.php` config will be as follow:
```

Comment:
Unfortunately, I don't understand the sentence or what the statement is. I will work out another proposal.

Reply:
Ok, I can close this PR for now so we can use yours :)

Reply:
> …so we can use yours…

Which one? :man_shrugging:

Reply:
When you have time to propose it, I believe you have better wording for it 👍

Reply:
Why did you delete the branch? All you had to do was explain what you wanted to say, then we would have found a suitable wording.

Reply:
That's already explained in the diff, while you can write a better word in separate PR per your said before. I just want to reduce my waiting list PR.

I can restore the branch, re-open PR, and commit your suggestion if you have better paragraph replacement.

Thank you.

Reply:
> I just want to reduce my waiting list PR.

But this does not solve the actual problem.

> …if you have better paragraph replacement.

I don't know what you are trying to say or what is happening there. I need an explanation in order to be able to create a suitable wording.

Reply:
You said you will work out another proposal, so I guess you had better wording already in mind :)

Reply:
_Help me out, otherwise I can't help you!_

What is happening? Why is `SetList::LAMINAS_SERVICEMANGER_40_AUTO_IMPORT` needed or better than `$rectorConfig->importNames()` or should both be used?

Reply:
The `SetList::LAMINAS_SERVICEMANGER_40_AUTO_IMPORT` include the base `LAMINAS_SERVICEMANGER_40` which auto import enabled. 

There is no better or worst, just a shortcut (1 statement vs multi).

 user can directly use `$rectorConfig->importNames();` after if they want.

Reply:
What is the recommended way? I think `$rectorConfig->importNames();`, as documented in the official documentation: https://getrector.com/documentation/import-names

So `SetList::LAMINAS_SERVICEMANGER_40_AUTO_IMPORT` is only _nice to have_ and because it corresponds to the coding standard of Laminas. Right?

Reply:
Yeah, `$rectorConfig->importNames()` probably better, but since Rector still version 0.*, anything can changed about the name.

The `SetList::LAMINAS_SERVICEMANGER_40_AUTO_IMPORT` is a shortcut in case user don't want to type multi statements in config, also safer as the config call already included.